<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AkashSwami" failures="1" tests="3" name="Submit Order Test" time="27.812" errors="0" timestamp="2024-09-28T17:55:19 IST">
  <testcase classname="akashswamiacademy.StandAloneTest2" name="submitorder" time="15.139">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at akashswamiacademy.StandAloneTest2.submitorder(StandAloneTest2.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- submitorder -->
  <testcase classname="akashswamiacademy.StandAloneTest2" name="submitorder" time="6.115"/>
  <testcase classname="akashswamiacademy.StandAloneTest2" name="VerifyOrderHistoryProduct" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyOrderHistoryProduct -->
</testsuite> <!-- Submit Order Test -->
